code/controllers/subsystem/vote.dm

```diff
@@ -248,6 +244,36 @@ SUBSYSTEM_DEF(vote)
 
 	return TRUE
 
+/**
+ * Checks if we can start a vote.
+ *
+ * * vote_initiator - The mob that initiated the vote.
+ * * forced - Whether we're forcing the vote to go through regardless of existing votes or other circumstances.
+ *
+ * Returns TRUE if we can start a vote, FALSE if we can't.
+ */
+/datum/controller/subsystem/vote/proc/can_vote_start(mob/vote_initiator, forced)
+	// Even if it's forced we can't vote before we're set up
+	if(!MC_RUNNING(init_stage))
+		if(vote_initiator)
+			to_chat(vote_initiator, span_warning("You cannot start vote now, the server is not done initializing."))
```

Comment:
i know this is copypasted from the SS initialization but let's fix it up while we're cleaning stuff

```suggestion
			to_chat(vote_initiator, span_warning("You cannot start a vote now, the server is not done initializing."))
```